README.md

```diff
@@ -18,7 +19,7 @@ Rufo supports all Ruby versions >= 2.2.4, but works most reliably with >= 2.3.**
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'rufo'
+gem 'rufo', '~> 0.1.0'
```

Comment:
Could you explain, please, why we force gem version?

Reply:
Personally, I'd like to pin a gem version as much as possible.

I know that the default README template generated via `bundle gem NAME` does not specify a version. I do not like this. I rather prefer Elixir's approach and encourage users to pin versions.

Having said that, I do not think we need `gem 'rufo'` at all in `README` for `rufo`, do we?
It's just a tool which you normally install via `gem install rufo`, right?

Should we drop the following parts from the readme?

```
Add this line to your application's Gemfile:

gem 'rufo', '~> 0.1.0'
And then execute:

$ bundle
```

:question:

Reply:
I think we can remove Gemfile part because rufo is a standalone tool.